02-peer-protocol.md

```diff
@@ -1119,6 +1232,8 @@ given in [BOLT #3](03-transactions.md#fee-calculation).
 The node _responsible_ for paying the Bitcoin fee:
   - SHOULD send `update_fee` to ensure the current fee rate is sufficient (by a
       significant margin) for timely processing of the commitment transaction.
+  - if `option_simplified_update` is negotiated:
+    - MUST NOT mix `update_fee` with other updates in the same commitment.
```

Comment:
does this mean any other updates, including other `update_fee` messages in same turn, or just `update_fee` vs all other update types?

Reply:
Good q! I think it means the former, but should be more explicit!